Sources/UIView.swift

```diff
@@ -185,17 +185,16 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
 
     open func insertSubview(_ view: UIView, at index: Int) {
         // XXX: This might not cover all cases yet. Managing these two hierarchies is complex...
-        let indexOfViewWeJustPushedForwardInArray = index + 1
         if index == 0 {
             layer.insertSublayer(view.layer, at: 0)
-        } else if let currentSubview = safeGetSubview(index: indexOfViewWeJustPushedForwardInArray) {
+        } else if let currentSubview = safeGetSubview(index: index) {
             layer.insertSublayer(view.layer, below: currentSubview.layer)
         } else {
             // The given index was greater than that of any existing subview, meaning:
             // We didn't replace any view. Just push the new layer to the end of the sublayers array.
             layer.addSublayer(view.layer)
         }
-
+        
```

Comment:
here is a random bunch of spaces. please adjust your Xcode settings so it removes whitespace at the end of whitespace only lines.

Reply:
@janek would be great if you could fix this before merging

Reply:
agh. weird, the setting seems to be in the correct state. Nevertheless, fixed - sorry.